### ISSUE TITLE
Thunderbird, Login with Microsoft Account gives an error "Blocked by X-Frame-Options Policy"

### ISSUE THREAD

**Issue by bvfalcon** [fastaddons/AI-Anywhere#1]
Hello and thank you for this plugin for Thunderbird!

I have a problem with login into OpenAI with Microsoft Account. Steps to reproduce:
1) On the first page "ChatGPT" with text "Welcome to ChatGPT. Log in with your OpenAI account to continue" click "Log in"
2) Click "Continue with Microsoft Account"
Got a page with header "Blocked by X-Frame-Options Policy" and full message

```
This page has an X-Frame-Options policy that prevents it from being loaded in this context.

Thunderbird prevented this page from loading in this context because the page has an X-Frame-Options policy that disallows it.
```

I have not found in Settings any option to solve this. For Firefox there is an [addon](https://www.reddit.com/r/firefox/comments/pd4fol/how_to_solve_issue_of_blocked_by_x_frame_options/) for this reason, but for Thunderbird there is no such addon.

Additional info: AI Anywhere for ChatGPT 1.4

**Comment by fastaddons** [comment]
Hello,
Thank you for reporting this.
They are making a lot of changes so things breaks a lot. I'll try to fix this today.

In any case, to fix this now, simply click the icon in the corner (to reopen popup in a new tab or window), there the login should work again.

**Comment by fastaddons** [comment]
Fixed in 1.6 (just released)
